core/tests/protractor_utils/general.js

```diff
@@ -72,7 +72,8 @@ var EDITOR_URL_SLICE = '/create/';
 var PLAYER_URL_SLICE = '/explore/';
 var LOGIN_URL_SUFFIX = '/_ah/login';
 var ADMIN_URL_SUFFIX = '/admin';
-var SCRIPTS_URL_SLICE = '/scripts/';
+// TODO(gvishal): update for prod as well.
```

Comment:
I noticed that in deploy.py, after processing all the files we execute all the tests, which would probably run on dev? is that ryt? Would we wanna run tests on a simulated prod server?